ebf935ff1176c844-Traceback (most recent call last):
  File "C:\Users\YQ\Desktop\小满海关数据爬取\step2.py", line 116, in get_data
    self.write2sql(sql)
  File "C:\Users\YQ\Desktop\小满海关数据爬取\step2.py", line 245, in write2sql
    cur.execute(sql)
  File "D:\Anaconda\lib\site-packages\pymysql\cursors.py", line 170, in execute
    result = self._query(query)
  File "D:\Anaconda\lib\site-packages\pymysql\cursors.py", line 328, in _query
    conn.query(q)
  File "D:\Anaconda\lib\site-packages\pymysql\connections.py", line 516, in query
    self._affected_rows = self._read_query_result(unbuffered=unbuffered)
  File "D:\Anaconda\lib\site-packages\pymysql\connections.py", line 727, in _read_query_result
    result.read()
  File "D:\Anaconda\lib\site-packages\pymysql\connections.py", line 1066, in read
    first_packet = self.connection._read_packet()
  File "D:\Anaconda\lib\site-packages\pymysql\connections.py", line 683, in _read_packet
    packet.check_error()
  File "D:\Anaconda\lib\site-packages\pymysql\protocol.py", line 220, in check_error
    err.raise_mysql_exception(self._data)
  File "D:\Anaconda\lib\site-packages\pymysql\err.py", line 109, in raise_mysql_exception
    raise errorclass(errno, errval)
pymysql.err.ProgrammingError: (1064, "You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 're looking for IN STOCK.  Wall 2 Wall Flooring #Inventory Matters!')' at line 1")
&dd77a4aef9b4847f-Traceback (most recent call last):
  File "C:\Users\YQ\Desktop\小满海关数据爬取\step2.py", line 116, in get_data
    self.write2sql(sql)
  File "C:\Users\YQ\Desktop\小满海关数据爬取\step2.py", line 245, in write2sql
    cur.execute(sql)
  File "D:\Anaconda\lib\site-packages\pymysql\cursors.py", line 170, in execute
    result = self._query(query)
  File "D:\Anaconda\lib\site-packages\pymysql\cursors.py", line 328, in _query
    conn.query(q)
  File "D:\Anaconda\lib\site-packages\pymysql\connections.py", line 516, in query
    self._affected_rows = self._read_query_result(unbuffered=unbuffered)
  File "D:\Anaconda\lib\site-packages\pymysql\connections.py", line 727, in _read_query_result
    result.read()
  File "D:\Anaconda\lib\site-packages\pymysql\connections.py", line 1066, in read
    first_packet = self.connection._read_packet()
  File "D:\Anaconda\lib\site-packages\pymysql\connections.py", line 683, in _read_packet
    packet.check_error()
  File "D:\Anaconda\lib\site-packages\pymysql\protocol.py", line 220, in check_error
    err.raise_mysql_exception(self._data)
  File "D:\Anaconda\lib\site-packages\pymysql\err.py", line 109, in raise_mysql_exception
    raise errorclass(errno, errval)
pymysql.err.ProgrammingError: (1064, "You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 'll love the way your new floor looks.')' at line 1")
&762072a4e13f19d7-Traceback (most recent call last):
  File "C:\Users\YQ\Desktop\小满海关数据爬取\step2.py", line 116, in get_data
    self.write2sql(sql)
  File "C:\Users\YQ\Desktop\小满海关数据爬取\step2.py", line 245, in write2sql
    cur.execute(sql)
  File "D:\Anaconda\lib\site-packages\pymysql\cursors.py", line 170, in execute
    result = self._query(query)
  File "D:\Anaconda\lib\site-packages\pymysql\cursors.py", line 328, in _query
    conn.query(q)
  File "D:\Anaconda\lib\site-packages\pymysql\connections.py", line 516, in query
    self._affected_rows = self._read_query_result(unbuffered=unbuffered)
  File "D:\Anaconda\lib\site-packages\pymysql\connections.py", line 727, in _read_query_result
    result.read()
  File "D:\Anaconda\lib\site-packages\pymysql\connections.py", line 1066, in read
    first_packet = self.connection._read_packet()
  File "D:\Anaconda\lib\site-packages\pymysql\connections.py", line 683, in _read_packet
    packet.check_error()
  File "D:\Anaconda\lib\site-packages\pymysql\protocol.py", line 220, in check_error
    err.raise_mysql_exception(self._data)
  File "D:\Anaconda\lib\site-packages\pymysql\err.py", line 109, in raise_mysql_exception
    raise errorclass(errno, errval)
pymysql.err.ProgrammingError: (1064, "You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 's is a full-service furniture store chain with locations in Southgate, MI, Novi,' at line 1")
&f95f190f6f06e50c-Traceback (most recent call last):
  File "C:\Users\YQ\Desktop\小满海关数据爬取\step2.py", line 116, in get_data
    self.write2sql(sql)
  File "C:\Users\YQ\Desktop\小满海关数据爬取\step2.py", line 245, in write2sql
    cur.execute(sql)
  File "D:\Anaconda\lib\site-packages\pymysql\cursors.py", line 170, in execute
    result = self._query(query)
  File "D:\Anaconda\lib\site-packages\pymysql\cursors.py", line 328, in _query
    conn.query(q)
  File "D:\Anaconda\lib\site-packages\pymysql\connections.py", line 516, in query
    self._affected_rows = self._read_query_result(unbuffered=unbuffered)
  File "D:\Anaconda\lib\site-packages\pymysql\connections.py", line 727, in _read_query_result
    result.read()
  File "D:\Anaconda\lib\site-packages\pymysql\connections.py", line 1066, in read
    first_packet = self.connection._read_packet()
  File "D:\Anaconda\lib\site-packages\pymysql\connections.py", line 683, in _read_packet
    packet.check_error()
  File "D:\Anaconda\lib\site-packages\pymysql\protocol.py", line 220, in check_error
    err.raise_mysql_exception(self._data)
  File "D:\Anaconda\lib\site-packages\pymysql\err.py", line 109, in raise_mysql_exception
    raise errorclass(errno, errval)
pymysql.err.ProgrammingError: (1064, "You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 's most TRUSTED source for professional flooring services. Servicing the Portland' at line 1")
&2ca83722f25cde8c-Traceback (most recent call last):
  File "C:\Users\YQ\Desktop\小满海关数据爬取\step2.py", line 116, in get_data
    self.write2sql(sql)
  File "C:\Users\YQ\Desktop\小满海关数据爬取\step2.py", line 245, in write2sql
    cur.execute(sql)
  File "D:\Anaconda\lib\site-packages\pymysql\cursors.py", line 170, in execute
    result = self._query(query)
  File "D:\Anaconda\lib\site-packages\pymysql\cursors.py", line 328, in _query
    conn.query(q)
  File "D:\Anaconda\lib\site-packages\pymysql\connections.py", line 516, in query
    self._affected_rows = self._read_query_result(unbuffered=unbuffered)
  File "D:\Anaconda\lib\site-packages\pymysql\connections.py", line 727, in _read_query_result
    result.read()
  File "D:\Anaconda\lib\site-packages\pymysql\connections.py", line 1066, in read
    first_packet = self.connection._read_packet()
  File "D:\Anaconda\lib\site-packages\pymysql\connections.py", line 683, in _read_packet
    packet.check_error()
  File "D:\Anaconda\lib\site-packages\pymysql\protocol.py", line 220, in check_error
    err.raise_mysql_exception(self._data)
  File "D:\Anaconda\lib\site-packages\pymysql\err.py", line 109, in raise_mysql_exception
    raise errorclass(errno, errval)
pymysql.err.ProgrammingError: (1064, "You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near 's comfortable home furniture. Shop for sofas, couches, recliners, chairs, tables' at line 1")
&